rfcs/xxxx-standard-gh-labels.md

```diff
@@ -0,0 +1,118 @@
+
+## Proposal
+A standard baseline of labels to be applied across the cds-snc github organization. This standard will be enforced through a github action on each repo, and will allow for a consistent way to identify issues across the organization and will allow for products, business units or specific products to extend this baseline with labels that fit their needs. This will also allow for CDS to ensure that we are respecting our requirements to communicate with people in their official language of choice.
+
+See https://github.com/cds-snc/covid-alert-server-labels for an example of what is being proposed to implement this solution. 
+
+The source of truth for these labels will reside in github and what is put forth in this document is just to start the discussion. The understanding is that if this Standard is accepted that these labels will be copied into github and maintenance and discussion will continue in those repositories. 
+
+### Baseline Labels
+This list of labels will be standardized across the organization and will be mandatory, they will apply high level groupings and statuses and will allow for a consistent way of talking about issues across projects 
+
+This consistent vocabulary will enable us to create reporting tools that can be applied to any project using the system and will provide a consistent view into the state of all CDS products. It should also reduce cognitive burden when onboarding to the project. 
+
+The list of proposed baseline labels can be seen in Appendix A
+
+### Product Specific Labels
+
+Each product will be allowed to apply their own unique set of labels alongside the baseline labels. 
+
+
+- ```
+  Name: High Priority | Haute priorité
+  Description: 
+  Colour: 
+- ```
+  Name: Medium Priority  | Priorité moyenne
+  Description:
+  Colour:
+- ```
+  Name: Low Priority | Faible priorité
+  Description:
+  Colour
+- ```
+  Name: Bug | Bogue
+  Description:
+  Colour:
+- ```
+  Name: Documentation
+  Description:
+  Colour:
+- ```
+  Name: Security | Sécurité
+  Description:
+  Colour:
+- ```
+  Name: Privacy | Vie privée
+  Description:
+  Colour:
+- ```
+  Name: Wont fix | Ne résoudra pas
+  Description:
+  Colour:
+
+
+### Label Requirements
+
+Every label is bound by the following requirement:  
+
+- A label must be bilingual in the format `en | fr`
+- A label must not use a colour used by a baseline label
+
+It is recommended that labels make sense for the product/project they are being applied to and that they are not relevant to only a very small number of issues but this isn’t mandatory.
+
+### Governance
+
+Governance surrounding labels will occur in Github repos, as labels are defined as json objects and are enforced through Github actions we can use Github to apply a level of lightweight automated workflows. Github's Code owner functionality will be used to ensure 
```

Comment:
final sentence cut off.

Reply:
Fixed, it was a sentence I meant to make shorter.